components/github-api-client/src/error.rs

```diff
@@ -45,6 +47,7 @@ impl fmt::Display for HubError {
                         code, response)
             }
             HubError::AppAuth(ref e) => format!("GitHub App Authentication error, {}", e),
+            HubError::BuilderCore(ref e) => format!("{}", e),
```

Comment:
```suggestion
            HubError::BuilderCore(ref e) => e.to_string(),
```
Similarly for the other `format!("{}", e),` arms

Reply:
There's a lot of this all over the place - nice to have for a different PR maybe.